apps/AppWithWearable/android/app/src/main/AndroidManifest.xml

```diff
@@ -55,6 +55,7 @@
     <uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
```

Comment:
Doesn't seem to be reliable on android.
And let's not ask for permissions right when the app opens.

https://github.com/BasedHardware/Friend/assets/18078879/c1edc2cc-3257-4250-9897-5187959df398